imports/wasi_snapshot_preview1/args.go

```diff
@@ -108,7 +108,8 @@ func argsSizesGetFn(ctx context.Context, mod api.Module, params []uint64) Errno
 	mem := mod.Memory()
 	resultArgc, resultArgvLen := uint32(params[0]), uint32(params[1])
 
-	// Write the Errno back to the stack
+	// argc and argv_len offsets are not necessarily sequential, so we have to
+	// write them independently.
 	if !mem.WriteUint32Le(ctx, resultArgc, uint32(len(sysCtx.Args()))) {
```

Comment:
ps this doesn't inline. I'll check if anything gets better by inlining manually in another commit

Reply:
difference is inside the error range. I think generally we'll only optimize when there's a WriteUint32Le inside a loop